lib/workers/repository/update/pr/changelog/bitbucket/source.ts

```diff
@@ -0,0 +1,29 @@
+import is from '@sindresorhus/is';
+import { parseUrl } from '../../../../../../util/url';
+import type { BranchUpgradeConfig } from '../../../../../types';
+import { ChangeLogSource } from '../source';
+
+export class BitbucketChangeLogSource extends ChangeLogSource {
+  constructor() {
+    super('bitbucket', 'bitbucket-tags');
+  }
+
+  getAPIBaseUrl(config: BranchUpgradeConfig): string {
+    const parsedUrl = parseUrl(config.sourceUrl);
+    if (is.nullOrUndefined(parsedUrl)) {
+      return '';
+    }
+    const protocol = parsedUrl.protocol;
+    const host = parsedUrl.host;
+    return `${protocol}//api.${host}/`;
```

Comment:
🤔 shouldn't this always point to `api.bitbucket.org`? there is no self-hosted bitbucket cloud version.

Reply:
Correct - let me simplify this :)